src/material/list/selection-list.spec.ts

```diff
@@ -612,6 +613,34 @@ describe('MatSelectionList without forms', () => {
       expect(option.classList).toContain('mat-2-line');
     });
 
+    it('should be able to reach list options that are indirect descendants', () => {
```

Comment:
Is it possible that this test name is not matching the actual assertion? (looks like it's a leftover from copying)

Reply:
ops, good catch. Fixed.